configure.ac

```diff
@@ -579,6 +592,9 @@ else
   if test x"$enable_module_generator" = x"yes"; then
     AC_MSG_ERROR([NUMS generator module is experimental. Use --enable-experimental to allow.])
   fi
+  if test x"$enable_module_frost" = x"yes"; then
+    AC_MSG_ERROR([FROST module is experimental. Use --enable-experimental to allow.])
+  fi
```

Comment:
nit: when we execute `./configure --enable-module-frost`, the following error appears:
```bash
configure: error: MuSig module is experimental. Use --enable-experimental to allow.
``` 
instead of:
```
configure: error: FROST module is experimental. Use --enable-experimental to allow.
```
----
To resolve this issue, we should reposition the FROST module check before the MuSig experimental module check (lines 486-488).

Reply:
Can you try again? I wasn't able to reproduce the issue. 

When I run `./configure --enable-module-frost` I get `configure: error: FROST module is experimental. Use --enable-experimental to allow.`.